### PR TITLE
Support section data from RST

### PR DESCRIPTION
**Pull request by roosre**
The import of section data (layers) from the RST via the lay-up provider is now also supported in the python front end by passing the RST stream to the lay-up provider.
In addition, the `layup_model_type` property was added to the CompositeModel so that the user can figure out from which source(s) the sections were loaded. It can be ACP, RST or MIXED.

New unit tests were added and the existing ones were updated if needed.

The documentation was adjusted as well.

### CODE REVIEW

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py [hunk 1]
```diff
@@ -71,21 +76,28 @@ def add_layup_info_to_mesh(
...
-    )
-
     # Set up the lay-up provider.
     # Reads the composite definition file and enriches the mesh
     # with the composite lay-up information.
     layup_provider = Operator("composite::layup_provider_operator")
     layup_provider.inputs.mesh(mesh)
-    layup_provider.inputs.data_sources(composite_data_sources)
+
+    composite_data_sources = _get_composite_data_sources_for_layup_provider(
```
Comment (roosre) [inline src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:94]: the lay-up file is now optional and so passed to the lay-up provider only if it is available.

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py [hunk 2]
```diff
@@ -101,6 +113,12 @@ def add_layup_info_to_mesh(
         unit_system = material_operators.result_info_provider
 
     layup_provider.inputs.unit_system(unit_system)
+
+    if rst_stream_provider:
```
Comment (roosre) [inline src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:117]: pass the stream of the RST to the lay-up provider if it is available. This check is needed because the old server of the the backend does not support it.

File: src/ansys/dpf/composites/layup_info/_layup_info.py [hunk 3]
```diff
@@ -371,6 +372,14 @@ def get_indexer_no_data_pointer(field: PropertyField) -> PropertyFieldIndexerSin
 
         self.mesh = mesh
         self.corner_nodes_by_element_type = _get_corner_nodes_by_element_type_array()
+        self.apdl_material_field = self.mesh.elements.materials_field
+
+        self.solver_material_to_dpf_id = {}
+        if solver_material_ids is not None:
```
Comment (roosre) [inline src/ansys/dpf/composites/layup_info/_layup_info.py:378]: Enhance the ElementInfo by extracting the mapdl material ID from the RST and convert it into the DPF material ID.

File: src/ansys/dpf/composites/layup_info/_layup_info.py [hunk 4]
```diff
@@ -415,6 +424,15 @@ def get_element_info(self, element_id: int) -> Optional[ElementInfo]:
             assert layer_data[0] + 1 == len(layer_data), "Invalid size of layer data"
             n_layers = layer_data[0]
             is_layered = True
+        elif self.solver_material_to_dpf_id:
```
Comment (roosre) [inline src/ansys/dpf/composites/layup_info/_layup_info.py:427]: Enhance the ElementInfo by extracting the mapdl material ID from the RST and convert it into the DPF material ID.

File: doc/source/index.rst [hunk 5]
```diff
@@ -46,11 +46,17 @@ to implement custom failure criteria and computation.
...
+MAPDL solution and a material file (matml) which was generated by Engineering Data.
+Lay-up files from ACP are optional and only required for some advanced operations. Refer to
+:ref:`Limitations`.
+
 Key features
 ''''''''''''
 
 Here are some key features of PyDPF Composites:
 
+* Post-processing of layered shell and solid elements, whether they were preprocessed by ACP or not.
```
Comment (PipKat) [inline doc/source/index.rst:59]:
```suggestion
* Postprocessing of layered shell and solid elements, whether they were preprocessed by ACP or not.
```

File: doc/source/index.rst [hunk 6]
```diff
@@ -46,11 +46,17 @@ to implement custom failure criteria and computation.
 
         Provides developer installation and usage information.
 
+The minimum set of inputs to run a post-processing for composites is the result file(s) of an
```
Comment (PipKat) [inline doc/source/index.rst:49]:
```suggestion
The minimum set of inputs to run a postprocessing for composites is the result files of an
```

File: doc/source/index.rst [hunk 7]
```diff
@@ -63,15 +69,28 @@ Here are some key features of PyDPF Composites:
...
 Limitations
 '''''''''''
-- Layered elements (section data) that have not been preprocessed with ACP are not supported.
-  For information on converting legacy models, see `Import of Legacy Mechanical APDL Composite Models`_
-  in the Ansys Help.
 - Only the Mechanical APDL solver is supported.
-
+- The following operators and features are only supported if the model was preprocessed with ACP
+  and if the corresponding lay-up definition file is passed to the :class:`.CompositeModel`:
+  - The evaluation of the failure criteria for sandwich (:class:`FaceSheetWrinklingCriterion`, :class:`ShearCrimpingCriterion`)
```
Comment (PipKat) [inline doc/source/index.rst:79]:
```suggestion
  - The evaluation of the failure criteria for sandwich (:class:`FaceSheetWrinklingCriterion`, :class:`ShearCrimpingCriterion`) classes
```

File: doc/source/index.rst [hunk 8]
```diff
@@ -63,15 +69,28 @@ Here are some key features of PyDPF Composites:
...
+    for solid elements. Layered shell elements are always supported.
+  - The computation of interlaminar normal stresses (s3) for layered shell elements.
+    Otherwise, s3 is zero. This also affects 3D failure criteria which use s3, such as Puck 3D.
+    This limitation does not affect (layered) solid elements.
+  - The support of variable materials and material fields. Without ACP, only the
+    temperature is considered for the evaluation of variable material properties.
+  - Global plies, ply-wise scoping, and ply-wise failure criteria. Layer-wise post-processing is
```
Comment (PipKat) [inline doc/source/index.rst:86]: I disagree with Vale. Do hyphenate "ply-wise" and add it to Vale's accept.txt file.

File: doc/source/index.rst [hunk 9]
```diff
@@ -63,15 +69,28 @@ Here are some key features of PyDPF Composites:
...
+    always supported thanks to the scripting
+  - Plotting results on the reference surface of a laminate.
+
+  Note: MAPDL models which have not been preprocessed with ACP can be converted. For more
```
Comment (PipKat) [inline doc/source/index.rst:90]:
```suggestion
  Note: MAPDL models that have not been preprocessed with ACP can be converted. For more
```

File: examples/008_assembly_example.py [hunk 10]
```diff
@@ -57,8 +57,8 @@
 irf_field.plot()
 
 # %%
-# Plot IRF
-# ~~~~~~~~
+# Plot IRF on The Reference Surface
```
Comment (PipKat) [inline examples/008_assembly_example.py:60]:
```suggestion
# Plot IRF on the reference surface
```

File: examples/011_rst_workflow.py [hunk 11]
```diff
@@ -0,0 +1,89 @@
+"""
+.. _rst_workflow_example:
+
+Failure Analysis of an MAPDL (RST) Model
```
Comment (PipKat) [inline examples/011_rst_workflow.py:4]:
```suggestion
Failure analysis of an MAPDL (RST) model
```

File: examples/011_rst_workflow.py [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+----------------------------------------
+
+The post-processing of a MAPDL (RST) model with layered elements which was not
+preprocessed by ACP is shown by this example. The difference between the RST
```
Comment (PipKat) [inline examples/011_rst_workflow.py:8]:
```suggestion
This example shows the postprocessing of an MAPDL (RST) model with layered elements that was not
preprocessed by ACP. The difference between the RST
```
Use active voice.

File: examples/011_rst_workflow.py [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+only and ACP based workflow is that `composite` of :class:`.ContinuousFiberCompositesFiles`
```
Comment (PipKat) [inline examples/011_rst_workflow.py:9]:
```suggestion
only and ACP-based workflow is that `composite` of the :class:`.ContinuousFiberCompositesFiles` class
```
Single backtick will only italicize the word. If this should be a code entity, use double backticks.

File: examples/011_rst_workflow.py [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+is empty and so the section data are automatically loaded from the RST file.
```
Comment (PipKat) [inline examples/011_rst_workflow.py:10]:
```suggestion
is empty, and so the section data are automatically loaded from the RST file.
```

File: examples/011_rst_workflow.py [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+
+The engineering data file (XML or ENGD) with the material properties is needed anyway.
+Otherwise, the material properties cannot be mapped. It is recommended
+to create it before solving the model. The engineering data file can be either generated with Ansys Workbench
+or ACP (Ansys Composite PrePost) standalone. Important: The material UUIDs in the engineering data file
+must be identical to the UUIDs in the Mechanical APDL (RST file).
+The material UUID can be set in Mechanical APDL with the command ``MP,UVID,<material index>,<value>``.
+
+This workflow is supported since version 2024 R2 (DPF Server version 8.0). A few advanced features are
+not supported with the RST onl workflow. Refer to Section :ref:`limitations` for details.
```
Comment (PipKat) [inline examples/011_rst_workflow.py:20]:
```suggestion
This workflow is supported in 2024 R2 (DPF Server version 8.0) and later. A few advanced features are
not supported with the RST onl workflow. For more information, see :ref:`limitations`.
```
Not sure about "RST only workflow". Should it be RST-only workflow?

File: examples/011_rst_workflow.py [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+
+# %%
+# Set up model and evaluate failures
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set up the composite model.
+
+composite_model = CompositeModel(composite_files_on_server, server)
+
+# %%
+# Evaluate failures for the entire model
```
Comment (PipKat) [inline examples/011_rst_workflow.py:77]:
```suggestion
# Evaluate failures for the entire model.
```

File: src/ansys/dpf/composites/_indexer.py [hunk 17]
```diff
@@ -110,9 +115,15 @@ class PropertyFieldIndexerNoDataPointerNoBoundsCheck:
...
+            self._indices = index_by_id.mapping
+            self._data: NDArray[np.int64] = np.array(field.data, dtype=np.int64)
+            self._max_id = index_by_id.max_id
+        else:
+            self._indices = np.array([], dtype=np.int64)
+            self._data = np.array([], dtype=np.int64)
+            self._max_id = 0
 
     def by_id(self, entity_id: int) -> Optional[np.int64]:
         """Get index by id.
```
Comment (PipKat) [inline src/ansys/dpf/composites/_indexer.py:129]:
```suggestion
        """Get index by ID.
```

File: src/ansys/dpf/composites/_indexer.py [hunk 18]
```diff
@@ -121,24 +132,34 @@ def by_id(self, entity_id: int) -> Optional[np.int64]:
...
+            )
+        else:
+            self._max_id = 0
+            self._indices = np.array([], dtype=np.int64)
+            self._data = np.array([], dtype=np.int64)
+            self._n_components = 0
+            self._data_pointer = np.array([], dtype=np.int64)
 
     def by_id(self, entity_id: int) -> Optional[NDArray[np.int64]]:
         """Get index by id.
```
Comment (PipKat) [inline src/ansys/dpf/composites/_indexer.py:165]:
```suggestion
        """Get index by ID.
```

File: src/ansys/dpf/composites/_indexer.py [hunk 19]
```diff
@@ -166,18 +187,26 @@ def by_id(self, entity_id: int) -> Optional[NDArray[np.int64]]:
 class FieldIndexerWithDataPointer:
     """Indexer for a dpf field with data pointer."""
```
Comment (PipKat) [inline src/ansys/dpf/composites/_indexer.py:188]:
```suggestion
    """Indexer for a DPF field with data pointer."""
```

File: src/ansys/dpf/composites/_indexer.py [hunk 20]
```diff
@@ -207,16 +236,21 @@ class PropertyFieldIndexerWithDataPointerNoBoundsCheck:
...
+                field._data_pointer, len(self._data) * self._n_components
+            )
+        else:
+            self._indices = np.array([], dtype=np.int64)
+            self._data = np.array([], dtype=np.int64)
+            self._n_components = 0
+            self._data_pointer = np.array([], dtype=np.int64)
 
     def by_id(self, entity_id: int) -> Optional[NDArray[np.int64]]:
         """Get index by id.
```
Comment (PipKat) [inline src/ansys/dpf/composites/_indexer.py:256]:
```suggestion
        """Get index by ID.
```

File: src/ansys/dpf/composites/data_sources.py [hunk 21]
```diff
@@ -149,14 +149,14 @@ def _get_rst_list(value: Union[list[_PATH], _PATH]) -> list[_PATH]:
...
 class CompositeDataSources:
     """Provides data sources related to the composite lay-up.
 
     Parameters
     ----------
     rst:
+        Result file. Currently only RST (MAPDL) is supported.
 
     material_support:
         NOTE: The 'material_support' is explicitly listed since it is currently not
```
Comment (PipKat) [inline src/ansys/dpf/composites/data_sources.py:162]:
```suggestion
        NOTE: The ``material_support`` parameter is explicitly listed because it is currently not
```
I would omit "NOTE:" if this is a parameter description.

File: src/ansys/dpf/composites/data_sources.py [hunk 22]
```diff
@@ -149,14 +149,14 @@ def _get_rst_list(value: Union[list[_PATH], _PATH]) -> list[_PATH]:
...
 @dataclass(frozen=True)
 class CompositeDataSources:
     """Provides data sources related to the composite lay-up.
 
     Parameters
     ----------
     rst:
+        Result file. Currently only RST (MAPDL) is supported.
 
     material_support:
```
Comment (PipKat) [inline src/ansys/dpf/composites/data_sources.py:161]:
```suggestion
    material_support :
```
Add a data type?

File: src/ansys/dpf/composites/data_sources.py [hunk 23]
```diff
@@ -165,6 +165,7 @@ class CompositeDataSources:
         material support from the first RST file. This is a workaround until the
         support for distributed RST is added.
     engineering_data:
```
Comment (PipKat) [inline src/ansys/dpf/composites/data_sources.py:167]:
```suggestion
    engineering_data :
```
Add a data type?

File: src/ansys/dpf/composites/data_sources.py [hunk 24]
```diff
@@ -165,6 +165,7 @@ class CompositeDataSources:
...
+        File with the material properties.
 
     old_composite_sources:
         This member is used to support assemblies in combination with the old
```
Comment (PipKat) [inline src/ansys/dpf/composites/data_sources.py:171]:
```suggestion
        Member used to support assemblies in combination with the old
```

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py [hunk 25]
```diff
@@ -11,12 +11,16 @@
 
 def _get_composite_data_sources_for_layup_provider(
     data_sources: CompositeDataSources, composite_definition_label: Optional[str] = None
-) -> DataSources:
+) -> Optional[DataSources]:
     """
     Prepare DataSources of composite for the DPF server.
 
     Ensure that DataSources object is compatible with the version of the layup provider.
```
Comment (PipKat) [inline src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:18]:
```suggestion
    Ensure that the ``DataSources`` object is compatible with the version of the lay-up provider.
```
Earlier, it seemed that lay-up was always hyphenated.

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py [hunk 26]
```diff
@@ -11,12 +11,16 @@
 
 def _get_composite_data_sources_for_layup_provider(
     data_sources: CompositeDataSources, composite_definition_label: Optional[str] = None
-) -> DataSources:
+) -> Optional[DataSources]:
     """
     Prepare DataSources of composite for the DPF server.
```
Comment (PipKat) [inline src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:16]: Should it be Prepare ``DataSources`` object of the composite for the DPF server?

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py [hunk 27]
```diff
@@ -71,21 +76,28 @@ def add_layup_info_to_mesh(
         dictionary key in the :attr:`.ContinuousFiberCompositesFiles.composite`
         attribute. This parameter is only required for assemblies.
         See the note about assemblies in the description for the :class:`.CompositeModel` class.
+    rst_stream_provider:
+        Pass RST stream to load section data directly from the RST file. Support with DPF 2024R2
+        (8.0) and later.
+
     Returns
     -------
     :
```
Comment (PipKat) [inline src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:85]: Add object type of returned object.

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py [hunk 28]
```diff
@@ -71,21 +76,28 @@ def add_layup_info_to_mesh(
         dictionary key in the :attr:`.ContinuousFiberCompositesFiles.composite`
         attribute. This parameter is only required for assemblies.
         See the note about assemblies in the description for the :class:`.CompositeModel` class.
+    rst_stream_provider:
```
Comment (PipKat) [inline src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:79]: In most libraries, there is a space before the colon and then a space and the data type. Is yours a bit different for a reason?

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py [hunk 29]
```diff
@@ -71,21 +76,28 @@ def add_layup_info_to_mesh(
...
+        Pass RST stream to load section data directly from the RST file. Support with DPF 2024R2
+        (8.0) and later.
```
Comment (PipKat) [inline src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:81]:
```suggestion
        Pass RST stream to load the section data directly from the RST file. This parameter is support in DPF 2024 R2
        (8.0) and later.
```

File: src/ansys/dpf/composites/layup_info/_layup_info.py [hunk 30]
```diff
@@ -60,6 +61,19 @@ def _get_analysis_ply(mesh: MeshedRegion, name: str, skip_check: bool = False) -
     return mesh.property_field(property_field_name)
 
 
+def _get_layup_model_context(layup_provider: dpf.Operator) -> int:
+    """Get the lay-up model context from the lay-up provider."""
+    return layup_provider.get_output(218, int)
+
+
+#  Note: must be in sync with the LayupModelContextTypeEnum in the C++ code
```
Comment (PipKat) [inline src/ansys/dpf/composites/layup_info/_layup_info.py:69]:
```suggestion
#  Note: Must be in sync with the ``LayupModelContextTypeEnum`` object in the C++ code.
```

File: tests/rst_only_workflow_test.py [hunk 31]
```diff
@@ -0,0 +1,109 @@
...
+from ansys.dpf.composites.layup_info.material_properties import MaterialProperty
+from ansys.dpf.composites.server_helpers import version_equal_or_later, version_older_than
+
+from .helper import Timer
+
+SEPARATOR = "::"
+
+
+def test_composite_model_with_rst_only(dpf_server, data_files, distributed_rst):
+    """Test features of the composite model whith sectin data from the RST file only."""
```
Comment (PipKat) [inline tests/rst_only_workflow_test.py:28]:
```suggestion
    """Test features of the composite model with section data from the RST file only."""
```

File: .github/workflows/ci_cd.yml [hunk 32]
```diff
@@ -13,7 +13,7 @@ on:
 
 env:
   MAIN_PYTHON_VERSION: '3.9'
-  CONTAINER_TAG: 'latest'
+  CONTAINER_TAG: 'mapdl_section_data'
```
Comment (roosre) [inline .github/workflows/ci_cd.yml:16]: Todo: revert before this PR will be completed.

File: src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py (hunk 28 above)
Comment (roosre) [reply src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py:79]: The type is visible in the function header directly. I think it is not needed here again.

File: tox.ini [hunk 33]
```diff
@@ -31,7 +31,7 @@ setenv =
         PYTEST_COV_APPEND_ARG = --cov-append
 commands =
     poetry install -E test
-    poetry run pytest --reruns 1 --license-server={env:LICENSE_SERVER:} --image-tag=latest {env:PYTEST_MARKERS:} {env:PYTEST_EXTRA_ARGS:} {env:PYTEST_COV_APPEND_ARG:} {posargs:-vv}
+    poetry run pytest --reruns 1 --license-server={env:LICENSE_SERVER:} --image-tag=mapdl_section_data {env:PYTEST_MARKERS:} {env:PYTEST_EXTRA_ARGS:} {env:PYTEST_COV_APPEND_ARG:} {posargs:-vv}
```
Comment (roosre) [inline tox.ini:34]: todo: revert this change

File: tests/conftest.py [hunk 34]
```diff
@@ -58,7 +58,7 @@ def pytest_addoption(parser: pytest.Parser) -> None:
     parser.addoption(
         DOCKER_IMAGE_TAG_KEY,
         action="store",
-        default="latest",
+        default="mapdl_section_data",
```
Comment (roosre) [inline tests/conftest.py:61]: todo: revert this change

File: doc/source/index.rst [hunk 35]
```diff
@@ -46,11 +46,17 @@ to implement custom failure criteria and computation.
 
         Provides developer installation and usage information.
 
+The minimum set of inputs to run a postprocessing for composites is the result files of an
+MAPDL solution and a material file (MATML) which was generated by Engineering Data.
+Lay-up files from ACP are optional and only required for some advanced operations. Refer to
+:ref:`Limitations`.
+
```
Comment (janvonrickenbach) [inline doc/source/index.rst:53]: Have you tried that a MatML file from engineering data is enought. I'm not sure if the material uuids are then written.

File: doc/source/index.rst [hunk 36]
```diff
@@ -46,11 +46,17 @@ to implement custom failure criteria and computation.
...
 Key features
 ''''''''''''
 
 Here are some key features of PyDPF Composites:
 
+* Postprocessing of layered shell and solid elements, whether they were preprocessed by ACP or not.
```
Comment (janvonrickenbach) [inline doc/source/index.rst:59]: As a user, without context this sentence can be confusing I think (What does it me if the model has not been preprocessed by ACP?).
Postprocessing of layered shell and solid elements. MAPDL models as well as models preprocessed with ACP are supported.

File: doc/source/index.rst [hunk 37]
```diff
@@ -63,15 +69,34 @@ Here are some key features of PyDPF Composites:
...
+- The following operators and features are only supported if the model was
+  preprocessed with ACP and if the corresponding lay-up definition file is passed to the :class:`.CompositeModel` class.
+
+  - The evaluation of the failure criteria for sandwich
+    (:class:`FaceSheetWrinklingCriterion <.failure_criteria.FaceSheetWrinklingCriterion>`,
+    :class:`ShearCrimpingCriterion <.failure_criteria.ShearCrimpingCriterion>`)
+    for solid elements. Layered shell elements are always supported.
+
+  - The computation of interlaminar normal stresses (s3) for layered shell elements.
+    Otherwise, s3 is zero. This also affects 3D failure criteria which use s3,
```
Comment (janvonrickenbach) [inline doc/source/index.rst:86]:
```suggestion
    Without ACP layup definitions, s3 is always zero. This also affects 3D failure criteria which use s3,
```

File: examples/011_rst_workflow.py [hunk 38]
```diff
@@ -0,0 +1,94 @@
...
+)
+from ansys.dpf.composites.server_helpers import connect_to_or_start_server
+
+# %%
+# Start a DPF server and copy the example files into the current working directory.
+server = connect_to_or_start_server()
+
+# %%
+# Get input files (RST and material.engd but skip the ACP layup file).
+composite_files_on_server = get_continuous_fiber_example_files(server, "shell", True)
```
Comment (janvonrickenbach) [inline examples/011_rst_workflow.py:55]: As mentioned above. I think we should ensure that the materials can be generated without mechanical and pyACP.

File: src/ansys/dpf/composites/layup_info/_layup_info.py [hunk 39]
```diff
@@ -371,6 +388,14 @@ def get_indexer_no_data_pointer(field: PropertyField) -> PropertyFieldIndexerSin
 
         self.mesh = mesh
         self.corner_nodes_by_element_type = _get_corner_nodes_by_element_type_array()
+        self.apdl_material_field = self.mesh.elements.materials_field
```
Comment (janvonrickenbach) [inline src/ansys/dpf/composites/layup_info/_layup_info.py:391]: Can you use the indexer to improve performance as for the other fields?

File: src/ansys/dpf/composites/layup_info/_layup_info.py (hunk 4 above)
Comment (janvonrickenbach) [reply src/ansys/dpf/composites/layup_info/_layup_info.py:443]: This function is quite performance sensitive. Could you test the performance of the new branch for a reasonably large model.

File: src/ansys/dpf/composites/server_helpers/_versions.py [hunk 40]
```diff
@@ -14,8 +14,15 @@ class _DpfVersionInfo:
 _DPF_VERSIONS: dict[str, _DpfVersionInfo] = {
     "5.0": _DpfVersionInfo("5.0", "2023 R1", "Initial release of DPF Composites."),
     "7.0": _DpfVersionInfo("7.0", "2024 R1 pre 0", "DPF Composites plugin with sub-operators."),
```
Comment (janvonrickenbach) [inline src/ansys/dpf/composites/server_helpers/_versions.py:16]: Formating looks a bit inconsistent

**Review by janvonrickenbach — APPROVED** [review]
